front/src/pages/settings/data-model/SettingsObjectDetail.tsx

```diff
@@ -103,6 +103,7 @@ export const SettingsObjectDetail = () => {
           title="Add Field"
           size="small"
           variant="secondary"
+          onClick={() => navigate(`./new-field/step1`)}
```

Comment:
The Step 1 page is only accessible if there are disabled fields:

```suggestion
          onClick={() =>
            navigate(
              disabledFieldItems.length
                ? './new-field/step1'
                : './new-field/step2',
            )
          }
```

We can implement it now or when we create the Step 2 page if you want.